src/installer/Directory.Build.targets

```diff
@@ -39,11 +32,8 @@
       <ProductVersion>$(ProductionVersion)$(ProductVersionSuffix)</ProductVersion>
 
       <SharedFrameworkNugetVersion>$(ProductVersion)</SharedFrameworkNugetVersion>
-
-      <RuntimeDepsRpmVersion Condition="'$(RuntimeDepsRpmVersion)'==''">$(ProductVersion)</RuntimeDepsRpmVersion>
-      <RuntimeDepsDebVersion Condition="'$(RuntimeDepsDebVersion)'==''">$(ProductVersion)</RuntimeDepsDebVersion>
-
       <NuGetVersion>$(SharedFrameworkNugetVersion)</NuGetVersion>
+      <InstallersRelativePath>Runtime/$(SharedFrameworkNugetVersion)/</InstallersRelativePath>
```

Comment:
nit: could inline the value at four call-sites at https://github.com/dotnet/runtime/blob/e5c7b036a094a420040c706b4fd275c2cf153a09/src/installer/publish/prepare-artifacts.proj#L98

Reply:
I hope that a big chunk of the publishing infra under installer can be removed in favor of Arcade's publishing infra. I need to create an issue for that to start the discussion (and understand the constraints). It's unclear to me why we are still doing so much of the publishing ourselves.